src/particleContainer/adapter/vectorization/RealVecDouble.h

```diff
@@ -430,8 +432,11 @@ class RealVec<double> {
 				const RealVec inv_24bits = inv * fnmadd(d, inv, set1(2.0)); 				//24bit, 1. N-R-Iteration
 				const RealVec inv_48bits = inv_24bits * fnmadd(d, inv_24bits, set1(2.0));	//48bit, 2. N-R-Iteration
 				const RealVec inv_prec = inv_48bits * fnmadd(d, inv_48bits, set1(2.0)); 	//96bit, 3. N-R-Iteration
```

Comment:
do we really need the third iteration here? IEEE anyways has only 52 bits for double precision (mantissa)

Reply:
also below

Reply:
That's the 256 bit version, it has not been changed. 
Yes it overshoots considerably, but the idea is to be on the safe side and guarantee IEEE precision. So in order to get >= 53 bits precision, one more iteration is needed. The problem is that the 12-bit rcp and rsqrt versions in AVX are good for iterating up to _float_ precision (>= 24 bits), but bad for iterating up to _double_ precision. That's why they upgraded the rcp and rsqrt in AVX512F to 14 bits (and even 28 bits in AVX512ER). 

I would prefer to keep it like this, because we don't know how those things potentially propagate in the more complex kernels such as dipole and quadrupole, which need to evaluate higher powers, such as r^(-5), r^(-6).

Reply:
ok